src/models/db/case.js

```diff
@@ -1,72 +1,134 @@
 'use strict'
 
-module.exports = CaseFactory
-
-const Container = require('constitute').Container
-const Model = require('five-bells-shared').Model
-const UriManager = require('../../lib/uri')
-const config = require('../../lib/config')
-const PersistentKnexModelMixin = require('five-bells-shared').PersistentKnexModelMixin
-const Validator = require('five-bells-shared/lib/validator')
 const knex = require('../../lib/knex').knex
-const moment = require('moment')
+const _ = require('lodash')
+const co = require('co')
+const assert = require('assert')
 
-CaseFactory.constitute = [UriManager, Validator, Container]
-function CaseFactory (uri, validator, container) {
-  class Case extends Model {
-    static convertFromExternal (data) {
-      // ID is optional on the incoming side
-      if (data.id) {
-        data.id = uri.parse(data.id, 'case').id.toLowerCase()
-      }
+const TABLE_NAME = 'N_CASES'
 
-      data.expires_at = new Date(data.expires_at)
+const CASE_STATUS_IDS = {
+  1: 'proposed',
+  2: 'rejected',
+  3: 'executed'
+}
 
-      return data
-    }
+const CASE_DESCRIPTIONS = _.invert(CASE_STATUS_IDS)
 
-    static convertToExternal (data) {
-      data.id = uri.make('case', data.id.toLowerCase())
-      data.notaries = [config.getIn(['server', 'base_uri'])]
-      data.expires_at = moment(data.expires_at).toISOString() // format('YYYY-MM-DD HH:mm:ss');
-      delete data.Notaries
-      if (!data.exec_cond_fulfillment) {
-        delete data.exec_cond_fulfillment
-      }
-      return data
-    }
+function knexOrTransaction (knex, options) {
```

Comment:
I renamed this to `getTransaction` in the ledger, maybe not the perfect name, but I think it's better than `knexOrTransaction`